### ISSUE TITLE
Create partner-bugzilla.redhat.com user for rw testing

### ISSUE THREAD

**Issue by abn** [python-bugzilla/python-bugzilla#52]
Created google group for this purpose. This user needs to be created an account on partner-bugzilla.redhat.com for enabling automated testing on travis.

Blocks: #42

**Comment by abn** [comment]
Account requested, waiting for confirmation email (hmm).

**Comment by crobinso** [comment]
you'll need to create the account on bugzilla.redhat.com, then wait for the next partner-bugzilla state sync, which might take a while. partner-bugzilla doesn't send any email so you can't create an account on it AFAIK.

the google group is completely private except to members, right?

**Comment by abn** [comment]
@crobinso yes it is completely private; there are two groups being used one for admin and the other for ci spam. Both are locked down.

**Comment by abn** [comment]
Did not know the accounts were synced over. Will register on bugzilla.redhat.com.

**Comment by crobinso** [comment]
Duplicate of #42
